components/main/main.component.tsx

```
@@ -1,19 +1,39 @@
+    name: 'Tari',
+    icon: '/icons/tari.png',
+  })
+
+  const [toNetwork, setToNetwork] = useState({
+    name: 'Ethereum',
+    icon: '/icons/eth.png',
+  })
+
   const { isConnected } = useAccount()
 
+  const handleNetworkSelect = (network: any, type: 'from' | 'to') => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)